context/app/static/js/components/Search/SearchWrapper.jsx

```diff
@@ -51,7 +51,7 @@ function makeTableComponent(resultFields, detailsUrlPrefix, idField) {
     const { hits } = props;
     /* eslint-disable no-underscore-dangle */
     return (
-      <table className="sk-table sk-table-striped" style={{ width: '100%' }}>
```

Comment:
Is the width setting being removed for a particular reason?

Reply:
It just got moved to the SCCS file to tidy things up.